src/modules/commander/failsafe/framework.cpp

```diff
@@ -222,16 +222,16 @@ void FailsafeBase::notifyUser(uint8_t user_intended_mode, Action action, Action
 				events::send<uint32_t>(
 					events::ID("commander_failsafe_enter_generic_warn"),
 				{events::Log::Warning, events::LogInternal::Warning},
-				"Failsafe warning triggered", mavlink_mode);
+				"Failsafe warning:", mavlink_mode);
 
 			} else {
 				/* EVENT
 				* @type append_health_and_arming_messages
 				*/
-				events::send<uint32_t, events::px4::enums::failsafe_action_t>(
+				events::send<uint32_t>(
 					events::ID("commander_failsafe_enter_generic"),
 				{events::Log::Critical, events::LogInternal::Warning},
-				"Failsafe, triggering {2}", mavlink_mode, failsafe_action);
+				"Failsafe activated: Autopilot disengaged", mavlink_mode);
```

Comment:
We want to gather the attention of the pilot: in the avionics world there are common sentences like "Autopilot degraded mode" or "Autopilot disengaged" that are really important for the pilot. this simple wording allows the pilot to understand that he needs to take an active control of the vehicle

Reply:
Why would you not tell the user what the action is that the failsafe will do?
I went through the test case of flying multicopter in position mode, then losing stick input, GPS and having low battery one after the other. The initial view when all of it happens quickly is just the first critical message which I think you mentioned @cmic0 is not ideal. We should consider showing multiple if they are not cleared or maybe the last one if that's not possible:
<img width="810" alt="Screenshot_7" src="https://github.com/PX4/PX4-Autopilot/assets/4668506/a93eeb38-f1a6-47e3-b4b4-565323ec1647">
But then when you press on it you get the entire rundown and what is the respective action:
<img width="679" alt="Screenshot_8" src="https://github.com/PX4/PX4-Autopilot/assets/4668506/3b88f83c-dfe6-4be4-9c47-88c650cc17c8">

with the suggested change here it gets less informative:
<img width="684" alt="Screenshot_13" src="https://github.com/PX4/PX4-Autopilot/assets/4668506/46b5dc94-a11d-4154-a76d-ad97ce9ae601">
In the example here it's not clear what autopilot disengaged means. How could the pilot take active control of the vehicle? You could argue that the active mode tells you what the action is but we consistently inform the user about what happened and what the action is. I'd also do that here.

Reply:
I added the information in 9e77790589ed18c54619ac21daac81983a189d43